en/mulesoft/modules/ROOT/pages/ms_composer_invocable_flows.adoc

```diff
@@ -0,0 +1,263 @@
+. <<activate-your-invocable-composer-flow, Activate your invocable Composer flow.>>
+
+[[configure-your-invocable-composer-flow]]
+=== Configure Your Invocable Composer Flow
+
+To configure an invocable Composer flow:
+
+. Access Composer.
+. xref:ms_composer_flows.adoc#create-a-new-flow[Create a new flow from scratch].
+. In the *What Should Start This Flow?* dialog, in the *Interaction* section, click *Invoke* to create an invocable flow. + 
+The *Trigger: Invoke this flow from an external system via HTTP POST Request* trigger appears. 
+. Optionally, add input fields. For example, you can add an `Opportunity ID` input that returns a value if Salesforce passes the value of an opportunity ID to the Composer flow. 
++
+To add input fields, click *Add Input Fields*. You can define input fields using the *Visual Editor* or the *Sample JSON Editor*. 
+
+* To define input fields using the *Visual Editor*:
+
+.. In the *Input Name* field, enter a unique name for the input field. The name must begin with a letter and cannot contain spaces.
+.. In the *Data Type* dropdown, select the data type you want to use for the input field. The following xref:ms_composer_custom_expression_editor.adoc#data-types[data types] are supported: `String`, `Number`, `Boolean`, `Date`, and `DateTime`.
+.. Optionally, add a *Sample Value* or *Help Text* to test the flow. 
+.. To add more input fields, click *Add Input*.
+.. Click *Apply*. +
+The number of input fields you added appears next to *Input Fields*, displaying one or more new input fields added to the flow.
+.. Optionally, to change the input fields, click *Add Input Fields* to open the *Visual Editor* to edit the input fields.
+
+* To define input fields using the Sample *JSON Editor*:
+
+.. In the *Example Request* window, paste or enter the JSON sample. 
++
+.Example: JSON sample in the *Sample JSON Editor* window
+[%collapsible]
+====
+[source,JSON]
+----
+{ 
+{    
+"name": "ben",
+"age": 100
+}
+}
+----
+====
++
+Unlike the *Visual Editor*, the *Sample JSON Editor* supports objects, arrays, and null values. 
++
+.Example: Complex JSON sample in the *Sample JSON Editor* window
+[%collapsible]
+====
+[source,JSON]
+----
+{
+"data": {
+"favoriteNumbers": [3, 7, 13, 17, 23]
+}
+}
+----
+====
++
+.. Click *Apply*. +
+The number of input fields you added appears next to *Input Fields*, displaying one or more new input fields added to the flow.
+.. Optionally, to change the input fields, click *Add Input Fields* to open the *Sample JSON Editor* to edit the input fields.
++
+. Complete the flow. For example, configure system actions for the flow, define variables, or create flow controls. For more information, refer to xref:ms_composer_flows.adoc[Build a Composer Flow]. 
+. Click *Save* to save your flow.
+
+If you need to delete an invocable step from your flow: 
+
+. Access the flow containing the invocable flow step that you want to delete. 
+. In the left corner of the step, click *…* > *Delete step*. +
```

Comment:
Can you rewrite this to remove "left," which isn't accessible? (I know that this isn't possible with a bad GUI.)

Reply:
I think this should be top right instead of left